src/index.html

```diff
@@ -11,12 +11,58 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    />
     <link
+      href="https://fonts.googleapis.com/css2?family=Jost:wght@400;500&family=Roboto:wght@500&display=swap"
       rel="stylesheet"
-      href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header_logo" href="/">
+        <img src="./images/logo.png" alt="moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#sumsung">Samsung</a>
```

Comment:
Don't use spaces in `<a>` tag's `href` property. The correct value for href should be '#samsung' instead of '#sumsung'.